Mathlib/Algebra/Group/Commute/Defs.lean

```diff
@@ -88,24 +88,20 @@ variable [Semigroup S] {a b c : S}
 "If `a` commutes with both `b` and `c`, then it commutes with their sum."]
 theorem mul_right (hab : Commute a b) (hac : Commute a c) : Commute a (b * c) :=
   SemiconjBy.mul_right hab hac
--- I think `ₓ` is necessary because of the `mul` vs `HMul` distinction
```

Comment:
All of these were comments about `#align` statements that have long since been removed.

Reply:
Indeed, thanks for spotting!